<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFeaturesGroups</class>
 <widget class="QDialog" name="FormFeaturesGroups">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группы параметров товара</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>172</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="Line" name="lineRefresh">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="QTreeView" name="treeViewGroups"/>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButtonAddGroup">
     <property name="toolTip">
      <string>Добавить группу для параметров</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/32/icons/32x32/add.png</normaloff>:/32/icons/32x32/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButtonEditGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Редактировать группу параметров</string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/32/icons/32x32/edit.png</normaloff>:/32/icons/32x32/edit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonDelGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Удалить выбранные группы</string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/32/icons/32x32/delete.png</normaloff>:/32/icons/32x32/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="toolButtonRefresh">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Обновить список групп параметров</string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/32/icons/32x32/refresh.png</normaloff>:/32/icons/32x32/refresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QToolButton" name="toolButtonGroupUp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Переместить группу выше</string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/32/icons/32x32/up.png</normaloff>:/32/icons/32x32/up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QToolButton" name="toolButtonGroupDown">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Переместить группу ниже </string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/32/icons/32x32/down.png</normaloff>:/32/icons/32x32/down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QToolButton" name="toolButtonSelect">
     <property name="toolTip">
      <string>Добавить группу для параметров</string>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
     <property name="icon">
      <iconset resource="../../res.qrc">
       <normaloff>:/64/icons/64x64/select.png</normaloff>:/64/icons/64x64/select.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
